src/components/AccountActivityCell.vue

```diff
@@ -87,7 +87,7 @@ const activityDescription = computed(() => {
     return 'Paid for'
   case 'InternalTransferEvent':
     if (props.activity.payload.kind === 'reward_block') {
-      return 'Beneficiary reward'
+      return 'Reward'
```

Comment:
I shortened some names, which effectively resolved the fight for space while still making sense.